pynucastro/networks/rate_collection.py

```diff
@@ -437,19 +437,21 @@ def _build_collection(self):
         self.reaclib_rates = []
         self.approx_rates = []
         self.derived_rates = []
+
         for r in self.rates:
             if isinstance(r, ApproximateRate):
                 self.approx_rates.append(r)
                 for cr in r.get_child_rates():
                     assert cr.chapter != "t"
-                    # child rates may be ReacLibRates or DerivedRates
-                    # make sure we don't double count
+                    # let Child ReacLibRates be the first in child_rates
+                    # and let DerivedRate be the second.
```

Comment:
I don't understand what this comment means now

Reply:
Oops, that was for previous versions but didn't work out well so I changed it. I'll get rid of those comments.